tests/Auth/Signup/Signup.spec.js

```diff
@@ -1,222 +1,143 @@
+            name: faker.person.firstName(), 
             lastName: faker.person.lastName(),
             email: faker.internet.email({firstName : 'aqa'}),
-            password: 'Qwertyuiopa'
+            password: 'Qwertyuiopa',
         }
-        await page.goto('/')
-
-        const signUpButton = page.locator('button:text("Sign up")')
-        await expect(signUpButton, "Registration link should be visible").toBeVisible()
-        await signUpButton.click()
-
-        const popup = page.locator('div.modal-dialog')
-        await expect(popup, "Registration in popup should be visible").toBeVisible()
-
-        const nameInput = popup.locator('input#signupName')
-        const lastNameInput = popup.locator('input#signupLastName')
-        const emailInput = popup.locator('input#signupEmail')
-        const passwordInput = popup.locator('input#signupPassword')
-        const reenterInput = popup.locator('input#signupRepeatPassword')
-        const registerButton = popup.locator('.btn-primary')
-
-        await nameInput.fill(user.firstName)
-        await lastNameInput.fill(user.lastName)
-        await emailInput.fill(user.email)
-        await passwordInput.fill(user.password)
-        await reenterInput.fill(user.password)
-        await expect(registerButton, "Register button should be disabled").toBeDisabled()
-
-        const passwordErrorMessage = popup.locator('div.invalid-feedback')
-        await expect(passwordErrorMessage, "Error message should be shown when user has entered password without numbers").toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter')
-        await expect(passwordInput, "Password input should have red border when user has entered invalid password").toHaveCSS('border-color', 'rgb(220, 53, 69)')
+        user.reenterPassword = user.password
+
+        signUpPopup = await welcomePage.openSignupPopup()
+        await signUpPopup.fillSignupForm(user)
+        await expect(signUpPopup.registerButton, "Register button should be disabled").toBeDisabled()
+        await expect(signUpPopup.errorMessage, "Error message should be shown when user has entered password without numbers").toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter')
+        await expect(signUpPopup.passwordInput).toHaveCSS('border-color', redBorder)
     });
 
     test('Should show error message when password do not match', async({page})=>{
         const user = {
-            firstName: faker.person.firstName(), 
+            name: faker.person.firstName(), 
             lastName: faker.person.lastName(),
             email: faker.internet.email({firstName : 'aqa'}),
             password: faker.internet.password({length: 9, prefix: 'Qw1'}),
-            reenterPassword: 'Qwertyuiopasdf1'
+            reenterPassword: 'Qwertyuiopasd1',
+            switchFocus: true
```

Comment:
switchFocus: true is not a part of user

Reply:
so better rename object

Reply:
Should I rename every instance of "user" in every other test, or only in this instance?